tests/test_groupedmessage.py

```diff
@@ -160,7 +150,9 @@ def test_messages(self):
         assert ret == InsertBatch([self.PROCESSED])
 
         assert not message_filter.should_drop(
-            self.__make_msg(0, 42, self.DELETE_MSG, [])
+            self.__make_msg(
+                0, 42, self.DELETE_MSG, [("table", "sentry_groupedmessage".encode())]
```

Comment:
I'm guessing that this was an oversight and this table header should actually be included?